src/components/HomePage/Login.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Container } from "reactstrap";
+function Features() {
+  return (
+    <Container>
```

Comment:
`Container` adds padding to the left and right. Keep in mind that this `<Login />` component is already nested under a `Container` in HomePage.js, so you will be padding this element twice.